conn.go

```diff
@@ -21,6 +21,8 @@ var (
 // ErrClosed is the error returned by calls on a closed connection.
 var ErrClosed = errors.New("dbus: connection closed by user")
 
+type q struct{}
```

Comment:
This feels like an unnecessary layer of indirection to me. Why not just use an empty struct literal?